### DISCUSSION TITLE
Cockpit 283

### DISCUSSION THREAD

**Discussion by maksymyefim** [cockpit-project/cockpit#18242] [Q&A] [answered]
Hello,
How to install the latest 283 version on RHEL 7.9?
I can get only 195 and no updates available then.

Thank you in advance.

**Comment by martinpitt** [comment, marked as answer]
Right, 195 is the latest available Cockpit on RHEL 7, later versions dropped RHEL 7 support. If you want a newer version, I suggest upgrading to RHEL 8 or 9.
